connectors/adform/connector.py

```diff
@@ -0,0 +1,381 @@
+"""This connector demonstrates how to fetch data from Adform API and upsert it into destination using the Fivetran Connector SDK.
+Supports querying data from Campaigns, Line Items, and Performance Data APIs.
+
+See the Technical Reference documentation (https://fivetran.com/docs/connectors/connector-sdk/technical-reference#update)
+and the Best Practices documentation (https://fivetran.com/docs/connectors/connector-sdk/best-practices) for details
+"""
+
+# For reading configuration from a JSON file
+import json
+
+# Import required classes from fivetran_connector_sdk
+from fivetran_connector_sdk import Connector
+
+# For enabling Logs in your connector code
+from fivetran_connector_sdk import Logging as log
+
+# For supporting Data operations like Upsert(), Update(), Delete() and checkpoint()
+from fivetran_connector_sdk import Operations as op
+
+# Import required libraries for API interactions
+import requests
```

Comment:
@fivertran-karunveluru 

The same here again, it will be exactly the same requirements.txt file as PR #260 from this comment: 
https://github.com/fivetran/fivetran_connector_sdk/pull/260/files#r2535556948